CoreUnitTests/.editorconfig

```diff
@@ -0,0 +1,218 @@
+# All Files
+[*]
```

Comment:
technically no, but it configures editors to be consistent for future contributers. As this is a new project its currently limited to this rather than affecting other existing projects.